src/synthesizer/parametric/sf_hist.py

```diff
@@ -383,147 +466,59 @@ def _sfr(self, age):
         return 0.0
 
 
-class ExponentiallyDeclining(Common):
-    """
-    An exponentially declining star formation history
-
-    Attributes:
-        tau (float)
-            The "stretch" parameter of the exponential.
-        initial_age (unyt_quantity)
-            The "start age" of the exponential, i.e. the age where
-            star formation is maximal.
-    """
-
-    def __init__(self, initial_age, tau, name=None):
-        """
-        Initialise the parent and this parametrisation of the SFH.
-
-        Args:
-            tau (unyt_quantity)
-                The "stretch" parameter of the exponential.
-            initial_age (unyt_quantity)
-                The "start age" of the exponential, i.e. the age where
-                star formation is maximal.
-        """
-
-        # Initialise the parent
-        Common.__init__(
-            self,
-            name="ExponentiallyDeclining",
-            tau=tau,
-            initial_age=initial_age,
-        )
-
-        # Set the model parameters
-        self.initial_age = initial_age.to("yr").value
-        self.tau = tau.to("yr").value
-
-    def _sfr(self, age):
-        """
-        Get the amount SFR weight in a single age bin.
-
-        Args:
-            age (float)
-                The age (in years) at which to evaluate the SFR.
-        """
-        if age < self.initial_age:
-            return np.exp(-1 * (self.initial_age - age) / self.tau)
-        return 0.0
-
-
-class DelayedExponentiallyDeclining(Common):
```

Comment:
Same as in the comment above, we should keep these. Much simpler for the user to have obvious naming signposting them. If they are just wrappers then that is fine.

Reply:
The delayed model was broken, it's now implemented properly as `DelayedExponential`.